lib/components/logIn.dart

```diff
@@ -39,6 +39,7 @@ class _LogInScreenState extends State<LogInScreen> {
 
   @override
   Widget build(BuildContext context) {
+    final MediaQueryData mediaQueryData = MediaQuery.of(context);
```

Comment:
Why call size again multiple times? 

do this 
```
final MediaQueryData size = MediaQuery.of(context).size;
```

width: size.width,
              height: size.height,